scripts/zones/Upper_Jeuno/npcs/_6s1.lua

```diff
@@ -1,106 +1,124 @@
------------------------------------
+----------------------------------------------------------------------------------------------------------------------------
```

Comment:
Grow up. This isn't 3rd grade. If you can't handle that and see it as some huge insult your skin is way too thin to be coding anything.